Dockerfile

```diff
@@ -1,5 +1,22 @@
-FROM alpine:3.7
+FROM openshift/origin-release:golang-1.9 AS build
```

Comment:
Interesting, is our own base image a new thing?

Reply:
It's used by a few other components and OpenShift CI for builds.

Reply:
Note... we don't actually ever build with this image as a base. It will be replaced with openshift3/ose-base in our process.
If you need this to also build in origin, then what you have is definitely the way to go. However, I am concerned about the golang-1.9 tag. That's not something I'm familiar with. We build openshift3/ose-base during the OCP build so that version will be what this image is built with.

Reply:
This is a multistage build. The first image is only used for the build.

https://docs.docker.com/develop/develop-images/multistage-build/

We have `FROM openshift/origin-base` later. Does the tooling support that?

Reply:
It doesn't yet, but this will be a guinea pig for that capability in OSBS.